.github/workflows/cocoapods.yaml

```diff
@@ -0,0 +1,20 @@
+name: Push to Cocoapods
+on:
+  workflow_dispatch:
+  push:
+    tags: ['*']
```

Comment:
better to use a release trigger? that way only actually publishing the release via github causes it to get published to cocoapods https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#release